server/bundler/loader.js

```diff
@@ -103,6 +113,7 @@ function splitTemplate( path, section ) {
 		'		controller.setSection( ' + JSON.stringify( section ) + ' )( context );',
 		'		if ( ! _loadedSections[ ' + JSON.stringify( section.module ) + ' ] ) {',
 		'			require( ' + JSON.stringify( section.module ) + ' )( controller.clientRouter );',
+		'			loadCSS( ' + JSON.stringify( section ) + ' );',
```

Comment:
Is there a chance that the JavaScript would load and render something before the CSS has loaded? If yes, do you think it would make sense to somehow hook the `unload` event on the `link` element?